accounts/abi/unpack_test.go

```diff
@@ -117,27 +117,12 @@ var unpackTests = []unpackTest{
 		want: int16(0),
 		err:  "abi: cannot unmarshal *big.Int in to int16",
 	},
-	{
-		def:  `[{"type": "bytes"}]`,
-		enc:  "000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000200100000000000000000000000000000000000000000000000000000000000000",
-		want: [32]byte{},
```

Comment:
You should create tests to validate what you expect the new behavior to be.

Reply:
modified the tests to test the new behaviour